packages/builder/src/builderStore/store/backend.js

```diff
@@ -60,6 +60,10 @@ export const getBackendUiStore = () => {
       },
       select: model =>
         store.update(state => {
+          model =
```

Comment:
Should we just:

A) Always pass a modelId to this function
B) Always pass a model object to this function